filetests/isa/x86/simd-arithmetic-run.clif

```diff
@@ -265,3 +265,17 @@ ebb0:
     return v4
 }
 ; run
+
+function %average_rounding_i16x8() -> b1 {
+ebb0:
+    v0 = vconst.i16x8 [0 0 0 1 42 19 -1 -1]
+    v1 = vconst.i16x8 [0 1 2 4 42 18 -1 0]
+    v2 = vconst.i16x8 [0 1 1 3 42 19 -1 -32768] ; -1 (0xffff) + 0 + 1 == -32768 (0x8000)
```

Comment:
Can the parser understand the hexadecimal syntax? If so, we should use it here, and then we wouldn't need the comment anymore.

Reply:
No, not yet for `i16`, `i32`, and `i64`; here's an issue to fix that: https://github.com/bytecodealliance/cranelift/issues/1401.